cpu/avr8_common/include/cpu.h

```diff
@@ -57,6 +57,11 @@ extern "C"
 #endif
 /** @} */
 
+/**
+ * @brief   CPU cycles per busy wait loop
+ */
+#define CPU_CYCLES_PER_LOOP (7)
```

Comment:
Did you guess this number or did you do some clever math? If so, can that math be documented?

Reply:
I incremented the number until the cycle time in the test app would match that of the µsec timer